app/src/main/java/com/group80/uoftinder/RecommendationView.java

```diff
@@ -0,0 +1,112 @@
+package com.group80.uoftinder;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+/**
+ * Public class that extends AppCompatActivity and implements RecViewInterface.
+ * This class displays the most compatible users to currentUser.
+ */
+public class RecommendationView extends AppCompatActivity implements RecViewInterface {
+    private final User currentUser;
+    private final RecommendationPresenter recPresenter;
+    private User displayedUser;
+
+    /**
+     * Initialize an instance of RecommendationView by passing the currentUser.
+     * @param currentUser is the current user.
+     */
+    public RecommendationView(User currentUser) {
+        this.currentUser = currentUser;
+        this.recPresenter = new RecommendationPresenter(this);
+        this.displayedUser = null;
+    }
+
+
+    // create variables for all elements that are displayed
+    ImageView profilePicture;
+    TextView name;
+    TextView gender;
+    TextView age;
+    Button noButton;
+    Button yesButton;
```

Comment:
Is it possible to move these into `onCreate` directly? Right now it seems like local variables are getting created outside of methods inside a class, so it's almost like instance variables but not declared that way formally? I would personally recommend either moving them into `onCreate` and make them actual local variables and/or declaring the ones that cannot be moved inside as private instance variables of the class!

Reply:
Yes, that was an oversight on my part! Will make them private instance variables of the class. Thanks Helen!